api/setup/setup.go

```diff
@@ -42,7 +43,20 @@ func Schemas(ctx context.Context, management *config.ManagementContext, schemas
 		}
 	}
 
-	return crdStore.AddSchemas(ctx, crdSchemas...)
+	if err := crdStore.AddSchemas(ctx, crdSchemas...); err != nil {
+		return err
+	}
+
+	for _, version := range crdVersions {
+		for key, schema := range schemas.SchemasForVersion(*version) {
+			crdSchemas = append(crdSchemas, schema)
```

Comment:
crdSchemas is just a temp structure to use on L46.  Here you can just do `schemas.Schema(managementschema.Version, "user")`